amethyst_tiles/Cargo.toml

```diff
@@ -33,6 +33,7 @@ lazy_static = "1.3"
 rayon = "1.0.2"
 bitintr = "0.3"
 glsl-layout = "0.3"
+err-derive = "0.1"
```

Comment:
Should we use 0.2? (the readme on `err-derive` does point to 0.1, but I think that's just forgotten to be updated)

Reply:
I just copied it from another place in the code. I can go through and update all of them if that makes sense for this PR

Reply:
ah don't worry about that, normally I'd do version bumps in a separate PR